pennylane/resource/second_quantization.py

```diff
@@ -180,6 +180,30 @@ def _flatten(self):
     def _unflatten(cls, data, metadata):
         return cls(*data, **dict(metadata))
 
+    @property
+    def lamb(self):
+        r"""Return the 1-Norm of the Hamiltonian."""
```

Comment:
Do we need to add a reference for how this is computed like in the first quantization case?

Reply:
Added description.